common/tekton/tasks/bumpversion.yaml

```diff
@@ -47,7 +47,7 @@ spec:
       name: scratch  
     workingDir: $(workspaces.gitrepos.path)/$(params.subdirectory)
   - name: bump-tag
-    image: quay.io/manuela/bumpversiontask:latest
+    image: quay.io/manuela/bumpversiontask:latest # review: is this supposed to be a static remote image path?
```

Comment:
When reviewing tasks to check for parameter usage, I found this statically defined remote image URL that seemed to be different in pattern to the http-ionic and pushprox task definitions. Please check if this is correct, and if it should remain unchanged.

Reply:
Definitely should *not* be static, well spotted.